src/CMakeLists.txt

```diff
@@ -646,11 +657,11 @@ add_dependencies(ceph-mon erasure_code_plugins)
 install(TARGETS ceph-mon DESTINATION bin)
 
 # OSD
-if(${HAVE_XFS})
+if(${HAVE_LIBXFS})
```

Comment:
@tchaikov agree! In the "CMake To Do" etherpad, making the syntax consistent is on the list, and will be a top priority when some of the more necessary functionality is working